apps/client/src/pages/WrapperPage.js

```diff
@@ -30,35 +67,35 @@ const WrapperPage = ({ checker, topic }) => {
         highlightActive
       >
         <StepByStepItem
-          active={!finishedLocation}
-          checked={finishedLocation}
+          active={() => { isActive('location') || isActive('address')}}
+          checked={() => isFinished('location', 'address')}
```

Comment:
I don't think this could actually work. `isFinished` uses `Array.includes` on the first argument only. You're passing two.